lms/envs/common.py

```diff
@@ -2323,6 +2323,7 @@
 }
 
 # E-Commerce API Configuration
+ECOMMERCE_EXTERNAL_URL_ROOT = None
```

Comment:
The API URL currently configured in stage and production is internal to the VPC (e.g. `stage-edx-ecommerce-internal`), meaning it's not publicly accessible. As part of the work we've done for refunds, the student support team will start receiving Zendesk tickets which include links to the ecommerce service from which they can approve or deny refunds. This new URL is meant to be something like `stage-edx-ecommerce.edx.org` as opposed to `stage-edx-ecommerce-internal.edx.org`.